packages/clarity-js/src/core/timeout.ts

```diff
@@ -2,7 +2,7 @@ import { Event } from "@clarity-types/data";
 import measure from "./measure";
 
 export function setTimeout(handler: (event?: Event | boolean) => void, timeout?: number, event?: Event): number {
-    return window.setTimeout(measure(handler), timeout, event);
+    return window.setTimeout(() => measure(handler)(event), timeout);
```

Comment:
This will change the order of operations and also the current usage is correct per the spec: https://developer.mozilla.org/en-US/docs/Web/API/setTimeout.